app/src/main/java/com/example/euriskocodechallenge/ui/home/viewmodel/MoreViewModel.kt

```diff
@@ -40,17 +31,30 @@ class MoreViewModel @Inject constructor(
     }
 
 
-    fun updateUser(fName: String, lName: String, imageBitmap: Bitmap) {
+    fun updateUser(fName: String, lName: String, imageString: String?) {
         viewModelScope.launch(Dispatchers.IO) {
             val userId = userDatabaseRepository.getLoggedInUserID()
-            userId.collect { id ->
-                userDatabaseRepository.getUserById(id).collectLatest { user ->
-                    user?.let {
-                        userDatabaseRepository.updateUser(it)
-                        UtilityFunctions.printLogs(Constants.TAG, "Updated User")
-                    }
+            userDatabaseRepository.getLoggedInUserID().collect {
+                var a: User? = null
```

Comment:
@martin371998 you should name your variables something more readable and relevant to their content